src/components/Card/Card.scss

```diff
@@ -21,69 +21,3 @@
 .card__category {
   color: $color-white;
 }
-
-.card__primary_color {
-  // note, for primary the bottom border is the ---100 variant
-  // all others are --400
-  border-bottom-color: $color-primary--100;
-
-  &.card__category {
-    background: $color-primary--700;
-  }
-}
-
-.card__jobs_color {
-  border-bottom-color: $color-jobs--400;
-
-  &.card__category {
-    background: $color-jobs--700;
-  }
-}
-
-.card__housing_color {
-  border-bottom-color: $color-housing--400;
-
-  &.card__category {
-    background: $color-housing--700;
-  }
-}
-
-.card__infrastructure_color {
-  border-bottom-color: $color-infrastructure--400;
-
-  &.card__category {
-    background: $color-infrastructure--700;
-  }
-}
-
-.card__liveability_color {
-  border-bottom-color: $color-liveability--400;
-
-  &.card__category {
-    background: $color-liveability--700;
-  }
-}
-
-.card__innovation_color {
-  border-bottom-color: $color-innovation--400;
-
-  &.card__category {
-    background: $color-innovation--700;
-  }
-}
-
-.card__planning_color {
-  border-bottom-color: $color-planning--400;
-
-  &.card__category {
-    background: $color-planning--700;
-  }
-}
-
-.card__extra_color {
-  border-bottom-color: $color-extra--400;
-
-  &.card__category {
-    background: $color-extra--700;
-  }
-}
```

Comment:
I ditched all of this because we know the colour value already in the component, so I just apply it directly as a style rather than this roundabout way.

The same approach won't work with the header category nav because you can't define hover states in styles.